src/main/java/org/cactoos/scalar/LengthOf.java

```diff
@@ -62,7 +90,7 @@ public LengthOf(final Input input) {
         }
     )
     public LengthOf(final Input input, final int max) {
-        super(() -> {
+        this(() -> {
             if (max == 0) {
                 throw new IllegalArgumentException(
                     "Cannot use a buffer limited to zero size"
```

Comment:
@dgroup I think this msg should be changed now

Reply:
@llorllale, sorry, didn't get what you mean.
The `max` field is buffer size, therefore if user pass the `0` he should receive the exception related to `zero-buffer-size`

Reply:
@dgroup sorry, my bad - I was not paying attention to the ctor this msg is in. Please disregard my comment.

Reply:
@llorllale, thank you, I think you missed it due to huge PR.
Sorry about that.